src/test/operators/join_test.hpp

```diff
@@ -9,6 +9,7 @@
 #include "gtest/gtest.h"
 
 #include "operators/abstract_join_operator.hpp"
+#include "operators/join_hash.hpp"
```

Comment:
That's not supposed to be here, isn't it?

Reply:
Yes it is, since this class is used for multi predicate join tests and was also adapted to create hash joins and sort-merge joins with secondary predicates.